pkg/util/log/log.go

```diff
@@ -47,13 +47,30 @@ func InitLogger(logFormat string, logLevel dslog.Level, buffered bool, rateLimit
 		logger = log.With(logger, "ts", log.DefaultTimestampUTC, "caller", log.Caller(5))
 	}
 	// Must put the level filter last for efficiency.
-	logger = level.NewFilter(logger, logLevel.Option)
+	Logger = newFilter(logger, logLevel)
```

Comment:
Is this meant to be an assignment to `logger`?

Reply:
Yes, good catch.  Got lost in merge conflicts.